Dictionary.WebApi/Repositories/ItemRepository.cs

```diff
@@ -35,5 +35,39 @@ public async Task DeleteItemsAsync(int id)
 
             await _dbConnection.ExecuteAsync(query, queryArguments);
         }
+
+        public async Task<Item?> GetItemByKeyAsync(string key)
+        {
+            string query = @"SELECT 
+                    id,
+                    key,
+                    content,
+                    expires_at,
+                    expiration_period
+                    FROM items WHERE key = @key";
+
+            var queryArguments = new { key };
+
+            return await _dbConnection.QueryFirstOrDefaultAsync<Item>(query, queryArguments);
+        }
+
+        public async Task InsertItemAsync(Item item)
+        {
+            var sql = "INSERT INTO items (key, content, expiration_period, expires_at) VALUES (@Key, @Content, @ExpirationPeriod, @ExpiresAt)";
+            await _dbConnection.ExecuteAsync(sql, item);
```

Comment:
when executing query you should put into arguments all values as in the other methods (new {Key, Content...}) not the Item itself